test/e2e/handler/rollback_test.go

```diff
@@ -92,7 +92,7 @@ interfaces:
     dhcp: true
     autoconf: true
     enabled: true
-`, nic))
+`, nic, nic, nic))
```

Comment:
can se use golang expression `%[1]s` so we don't have to repeat nic ?

Reply:
Done in https://github.com/nmstate/kubernetes-nmstate/pull/1225/files